src/gebt_poc/static_beam_element.cpp

```diff
@@ -0,0 +1,253 @@
+#include "src/gebt_poc/static_beam_element.h"
+
+#include <KokkosBlas.hpp>
+
+#include "src/utilities/log.h"
+
+namespace openturbine::gebt_poc {
+
+StaticBeamLinearizationParameters::StaticBeamLinearizationParameters()
+    : position_vectors_(Kokkos::View<double[35]>("position_vectors")),
+      stiffness_matrix_(StiffnessMatrix(gen_alpha_solver::create_matrix({
+          {1., 2., 3., 4., 5., 6.},       // row 1
+          {2., 4., 6., 8., 10., 12.},     // row 2
+          {3., 6., 9., 12., 15., 18.},    // row 3
+          {4., 8., 12., 16., 20., 24.},   // row 4
+          {5., 10., 15., 20., 25., 30.},  // row 5
+          {6., 12., 18., 24., 30., 36.}   // row 6
+      }))),
+      quadrature_(UserDefinedQuadrature(
+          std::vector<double>{
+              -0.9491079123427585,  // point 1
+              -0.7415311855993945,  // point 2
+              -0.4058451513773972,  // point 3
+              0.,                   // point 4
+              0.4058451513773972,   // point 5
+              0.7415311855993945,   // point 6
+              0.9491079123427585    // point 7
+          },
+          std::vector<double>{
+              0.1294849661688697,  // weight 1
+              0.2797053914892766,  // weight 2
+              0.3818300505051189,  // weight 3
+              0.4179591836734694,  // weight 4
+              0.3818300505051189,  // weight 5
+              0.2797053914892766,  // weight 6
+              0.1294849661688697   // weight 7
+          }
+      )) {
+    // Define the position vectors for the 5 node beam element
+    auto populate_position_vector = KOKKOS_LAMBDA(size_t) {
```

Comment:
Would it make sense to have this as a 2D view to match the refactor State class?